packages/compass-smoke-tests/src/installers/linux-tar.ts

```diff
@@ -18,7 +18,14 @@ export function installLinuxTar({
   execute('tar', ['-xzvf', filepath, '-C', sandboxPath]);
 
   // Check that the executable will run without being quarantined or similar
-  execute('xvfb-run', [path.resolve(appPath, appName), '--version']);
+  // Passing --no-sandbox because RHEL and Rocky usually run as root and --disable-gpu to avoid warnings
+  // (see compass-e2e-tests/helpers/chrome-startup-flags.ts for details)
+  execute('xvfb-run', [
+    path.resolve(appPath, appName),
+    '--version',
+    '--no-sandbox',
+    '--disable-gpu',
+  ]);
```

Comment:
Should we drop these changes, as we're no longer including the `linux_tar` in the workflow at all (and therefore not running this)

Reply:
We need those changes in order to run it on linux anyway, but I can leave it for whoever picks up the ticket to add it back

Reply:
I've removed it. The next person can figure it out again :)